client/src/fabric/tensegrity-fabric.ts

```diff
@@ -247,7 +261,42 @@ export class TensegrityFabric {
 
     public iterate(ticks: number): boolean {
         this.disposeOfGeometry()
-        return this.exports.iterate(ticks)
+        const changeHappened = this.exports.iterate(ticks)
+        if (!changeHappened) {
+            return false
+        }
+        const growth = this.growth
```

Comment:
```suggestion
        const {growth} = this
```

Or did we disallow that in TSlint?

Reply:
How is this better?

Reply:
fewer characters = better :) 

but maybe it's just a matter of preference. i don't feel v strongly about this